src/popover/popover-default-animation.jsx

```diff
@@ -0,0 +1,140 @@
+import Transitions from '../styles/transitions';
+import React from 'react';
+import PropTypes from '../utils/prop-types';
+import StylePropable from '../mixins/style-propable';
+import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
+import ThemeManager from '../styles/theme-manager';
+import Paper from '../paper';
+
+const PopoverAnimation = React.createClass({
+  mixins: [StylePropable],
+
+  propTypes: {
+    children: React.PropTypes.node,
+    className: React.PropTypes.string,
+    closing: React.PropTypes.bool,
+    open: React.PropTypes.bool.isRequired,
+    style: React.PropTypes.object,
+    targetOrigin: PropTypes.origin,
+    zDepth: PropTypes.zDepth,
+  },
+
+  getInitialState() {
+    return {
+      muiTheme: this.context.muiTheme ? this.context.muiTheme : ThemeManager.getMuiTheme(DefaultRawTheme),
+      mounted: false,
+    };
+  },
+
+  //for passing default theme context to children
+  childContextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  getChildContext() {
+    return {
+      muiTheme: this.state.muiTheme,
+    };
+  },
+
+  getDefaultProps() {
+    return {
+      style: {},
+      zDepth: 1,
+    };
+  },
+
+  componentWillMount() {
+    setTimeout(() => this.setState({mounted:true}), 0);
+  },
+
+  componentWillReceiveProps(nextProps, nextContext) {
+    let newMuiTheme = nextContext.muiTheme ? nextContext.muiTheme : this.state.muiTheme;
+
+    this.setState({
+      mounted:nextProps.open && !nextProps.closing,
+      muiTheme:newMuiTheme,
+    });
+  },
+
+  componentWillUnmount() {
```

Comment:
I think that we can remove this method. The Popover wait the end of the animation to unmount.